bouncerscript/task.py

```diff
@@ -97,7 +97,10 @@ def check_product_names_match_aliases(context):
 def check_product_names_match_nightly_locations(context):
     """Double check that nightly products are as expected"""
     products = context.task["payload"]["bouncer_products"]
-    if sorted(products) != sorted(BOUNCER_PATH_REGEXES_PER_PRODUCT.keys()):
+    valid_sets = []
+    for product_set in BOUNCER_PATH_REGEXES_PER_PRODUCT:
+        valid_sets.append(sorted(product_set.keys()))
+    if not sorted(products) in valid_sets:
```

Comment:
Nit: For some reason it seems more explicit to me to be `if sorted(products) not in valid_sets` but either is fine.